docs/sriov.md

```diff
@@ -32,7 +94,7 @@ $ find /sys -name *vfs*
 /sys/devices/pci0000:00/0000:00:09.0/0000:05:00.1/sriov_totalvfs
 ```
 
-Kubevirt will use `vfio` userspace driver to pass through PCI devices into
+KubeVirt will use `vfio` userspace driver to pass through PCI devices into
```

Comment:
Could you please explain somewhere in this doc why we use vfio instead of the default mode and why is it important for VMs. I'd appreciate if you linked or wrote documentation on how the attachment of VFIO device works in KubeVirt.

Reply:
I'm not completely sure what's the `default mode`, but I guess there can be a link to VFIO description somewhere to[1] or/and [2]

[1] https://www.kernel.org/doc/Documentation/vfio.txt
[2] https://www.linux-kvm.org/images/b/b4/2012-forum-VFIO.pdf